parts/linux/cloud-init/artifacts/ci-syslog-watcher.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Restarts ci-syslog service
+
+[Service]
+Type=oneshot
+ExecStart=/bin/systemctl restart ci-syslog.service
```

Comment:
is there a reason to do this?

make ci-syslog a oneshot and move the path trigger to the same service?